### ISSUE TITLE
[Bug]: windows mysql install fails

### ISSUE THREAD

**Issue by WizarDave** [beyondcode/herd-community#1221]
### Platform

Windows

### Operating system version

Windows 11

### System architecture

Windows

### Herd Version

1.13.0

### PHP Version

PHP 8.3

### Bug description

Fresh install of Herd. Create my first laravel site. MySQL connection fails. SQLite seems to work.

`        ✓ 53 modules transformed.
rendering chunks...
computing gzip size...
public/build/manifest.json             0.27 kB │ gzip:  0.15 kB
public/build/assets/app-C5xzoV4S.css  55.07 kB │ gzip:  9.18 kB
public/build/assets/app-Xaw6OIO1.js   34.85 kB │ gzip: 14.03 kB
✓ built in 14.15s

   Illuminate\Database\QueryException

  SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it (Connection: mysql, SQL: select exists (select 1 from information_schema.tables where table_schema = 'ei' and table_name = 'migrations' and table_type in ('BASE TABLE', 'SYSTEM VERSIONED')) as `exists`)

  at vendor\laravel\framework\src\Illuminate\Database\Connection.php:825
    821▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    822▕                 );
    823▕             }
    824▕
  ➜ 825▕             throw new QueryException(
    826▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    827▕             );
    828▕         }
    829▕     }

  1   vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:66
      PDOException::("SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it")

  2   vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:66
      PDO::__construct("mysql:host=127.0.0.1;port=3306;dbname=ei", "root", Object(SensitiveParameterValue), [])

   INFO  Livewire scaffolding installed successfully.`

### Steps to reproduce

On Windows 11

- Install Herd
- From Powershell (administrator): laravel new ei
- The error is received during install.

**Comment by mpociot** [comment]
@WizarDave are you using Herd Pro?
Herd Pro allows you to create and start database services like MySQL, Redis, etc.

**Comment by WizarDave** [comment]
This was from the Laravel installer. I expected it to ask for the mysql credentials, like other installers I've used, but it doesn't so it throws the error. Probably just an oversight on the installer programmers. ???
I've used Herd, but only with SQLite since I don't have any other options without upgrading to PRO. That installs without problems.

**Comment by mpociot** [comment]
Ah okay, then the error is coming from Laravel trying to connect to a non-existing database server instance and it's unrelated to Herd itself.
Either switch to SQLite or setup a MySQL server locally to fix this.

**Comment by WizarDave** [comment]
Yes, it can't connect without credentials. I can install with SQLite. But my point is that if the install script does not ask for credentials, it will always fail when using anything other than SQLite. The install script should ask for credentials when using other dBs.
